core/src/validator.rs

```diff
@@ -980,6 +982,18 @@ impl Validator {
             &connection_cache,
         );
 
+        let enable_quic_servers = if genesis_config.cluster_type == ClusterType::MainnetBeta {
+            config.enable_quic_servers
+        } else {
+            if config.enable_quic_servers {
```

Comment:
Should this be a warning when enable_quic_servers is false? For 'true', this check is noop anyway.

Reply:
the arg takes no values, so passing it anywhere but MB is a noop.  idea was to tell the operator "you did a thing that is superfluous"

Reply:
ah ok. makes sense.